alpha: 0.1
beta:  2.0
kappa: 0.0

noise_std:
  process:
    acc:
      x: 0.05
      y: 0.05
      z: 0.05
    w:
      x: 0.05
      y: 0.05
      z: 0.05
    acc_bias:
      x: 0.01
      y: 0.01
      z: 0.01
